### ISSUE TITLE
Clipboard not working when using --xwayland

### ISSUE THREAD

**Issue by CoolnsX** [mviereck/x11docker#512]
Hey, thanks for this script to make my life easier. 

I am having issues with clipboard not working whenever I run x11docker with --xwayland, I need it because the electron app does not play nicely without gpu even with --gpu args passed, but this break the clipboard sharing (bi-directional) functionality, so currently I am using the shared "file" workaround in which I saved clipboard contents in a file which is accessible from both container and host.

**Comment by mviereck** [comment]
Does clipboard sharing work e.g. with `Xephyr`?

**Comment by CoolnsX** [comment]
Yeah, clipboard works with xephyr, but I wanted to use xwayland, since the GPU acceleration is better there.

**Comment by mviereck** [comment]
Odd that it works in Xephyr, but not in Xwayland.
Just tried to reproduce and found that since my recent update to Debian bookworm the weston window with Xwayland does not show Xwayland at all, however, no error appears. So I cannot check the clipboard immediately.
I'll have to check and fix that first before I can check the clipboard issue.

**Comment by CoolnsX** [comment]
I use hyprland (Wayland compositor) and it has support for xwayland, that's what I use for running my custom docker image based on debian with xfce.
